styles/index.css

```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Lexend+Exa:wght@200;300;400;800;900&display=swap');
+
+body {
+    background-image: url('/papers.co-ag84-google-lollipop-march-mountain-background-29-wallpaper.jpg');
+    font-family: 'Bebas Neue', sans-serif;
+    background-color: #F5CAC3;
+
```

Comment:
for your background you could add 
`background-repeat: no-repeat;` so the the image wont repeat then add a width and height